apps/07jquery-tic-tac-toe/script.js

```diff
@@ -1,5 +1,85 @@
 'use strict';
 
-$(document).on('ready', function() {
-  // Put app logic in here
+$(document).ready(function() {
+
+    var $button = $("#clear")
+    var $datacell = $('[data-cell]');
+    var playerTurn = "X";
+    var $cell0 = $('[data-cell="0"]');
+    var $cell1 = $('[data-cell="1"]');
+    var $cell2 = $('[data-cell="2"]');
+    var $cell3 = $('[data-cell="3"]');
+    var $cell4 = $('[data-cell="4"]');
+    var $cell5 = $('[data-cell="5"]');
+    var $cell6 = $('[data-cell="6"]');
+    var $cell7 = $('[data-cell="7"]');
+    var $cell8 = $('[data-cell="8"]');
+
+    $datacell.click(function() {
+        $(this).text(playerTurn);
+        checkForWin();
+        playerTurn = (playerTurn == 'X') ? 'O' : 'X';
+    });
+
+    $button.click(function() {
+        $cell0 = $('[data-cell="0"]').text("");
+        $cell1 = $('[data-cell="1"]').text("");
```

Comment:
Since you already declared the [data-cell] jquery objects earlier, You can short-hand this code like so:

`$cell1 = $cell1.text();`

You can also use a for loop